varya/assets/sass/components/header/_header-branding.scss

```diff
@@ -42,3 +42,21 @@
 	font-family: var(--branding--description--font-family);
 	font-size: var(--branding--description--font-size);
 }
+
+a {
+	text-decoration: none;
+	border-bottom: 1px solid var(--global--color-secondary);
+}
```

Comment:
@jffng This is a tricky one. At first, I was thinking of moving it to `_reset.scss` where there’s already an `a {}` selector, but the bigger issue is that if we leave this in the Varya system, then every child theme that _doesn’t_ need an underline on anchor links will have to overwrite this (even when using the “Vanilla” variables). 

Wondering if there’s a way to have some styles like this one exist outside of the system in a way that allows child themes to not have to overwrite. 

This might also be minor enough that it’s not worth over-thinking in which case, I’d suggest just moving it to `_reset.scss` and undoing it in the Vanilla variables starter stylesheet. 

Cc: @kjellr

Reply:
Per our convo in slack, I think moving it to `_reset.scss` makes sense in this case.